File: templates/hexagonal-spring-boot/rest-adapter/src/test/java/packageName/rest/ExamplePoetryRestAdapterApplication.java
package packageName.rest;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.test.mock.mockito.MockBean;
import org.springframework.context.annotation.ComponentScan;
import packageName.domain.port.RequestExample;

@SpringBootApplication
@ComponentScan(basePackages = "packageName")
public class ExamplePoetryRestAdapterApplication {

  public static void main(String[] args) {
    SpringApplication.run(ExamplePoetryRestAdapterApplication.class, args);
  }

  @MockBean
  private RequestExample requestExample;
}

File: templates/hexagonal-spring-boot-kotlin/rest-adapter/src/test/kotlin/packageName/rest/ExamplePoetryRestAdapterApplication.kt
package packageName.rest

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.test.mock.mockito.MockBean
import org.springframework.context.annotation.ComponentScan
import packageName.domain.port.RequestExample

@SpringBootApplication
@ComponentScan(basePackages = ["packageName"])
class ExamplePoetryRestAdapterApplication {

    @MockBean
    private lateinit var requestExample: RequestExample
}

fun main(args: Array<String>) {
    SpringApplication.run(ExamplePoetryRestAdapterApplication::class.java, *args)
}
